cmd/api/routers/routers.go

```diff
@@ -110,6 +118,34 @@ func (c *Controller) GetNamespacedCoreResources(context *gin.Context) {
 	context.JSON(http.StatusOK, NewList(resources))
 }
 
+// Livez returns current server configuration as we don't have a clear deadlock indicator
+func (c *Controller) Livez(context *gin.Context) {
+
+	observabilityConfig := os.Getenv(observability.OtelStartEnvVar)
+	if observabilityConfig == "" {
+		observabilityConfig = "disabled"
+	}
+
+	context.JSON(http.StatusOK, gin.H{
+		"code":           http.StatusOK,
+		"ginMode":        gin.Mode(),
+		"authCacheTTL":   c.CacheConfiguration.Authorized,
+		"unAuthCacheTTL": c.CacheConfiguration.Unauthorized,
+		"openTelemetry":  observabilityConfig,
+		"message":        "healthy",
+	})
+}
+
+// Readyz checks Database connection
+func (c *Controller) Readyz(context *gin.Context) {
+	err := c.Database.Ping(context)
+	if err != nil {
+		abort.Abort(context, err.Error(), http.StatusServiceUnavailable)
+		return
+	}
```

Comment:
Is aborting what the code should be doing here? Can't you just set the context to `http.StatusServiceUnavailable` with an appropriate message?

Reply:
I'm calling `Abort()` because it's a utility defined [here](https://github.com/maruiz93/kubearchive/blob/main/cmd/api/abort/abort.go) that is used in other places of the API (endpoints and middleware) to save a couple of lines. It's just a shortcut to do these 3 steps:
* Log the error message
* Set the JSON output with a specific `message` and the desired http status
* abort the context (which I think it's expected based on go context management) 

So I think that we are good with the abort right here.